src/arbitrum/epoch-based-managed-vaults/stratagies/interfaces/IY2KVault.sol

```diff
@@ -0,0 +1,25 @@
+// SPDX-License-Identifier: MIT
+pragma solidity 0.8.17;
+
+interface IY2KVault {
+
+    /// @dev See {IERC1155-balanceOf}
+    /// @dev Returns the amount of tokens of token type `id` owned by `account`
+    /// @param account - address of the account to check the balance of
+    /// @param id - uint256 of the token type to check the balance of
+    function balanceOf(address account, uint256 id) external view returns (uint256);
+
+    /// @dev Mints Vault shares to receiver by depositing exact amount of assets
+    /// @param _assets - uint256 representing how many assets the user wants to deposit, a fee will be taken from this value
+    /// @param _receiver - address of the receiver of the assets provided by this function, that represent the ownership of the deposited asset
+    /// @param _id - uint256 in UNIX timestamp, representing the end date of the epoch. Example: Epoch ends in 30th June 2022 at 00h 00min 00sec: 1654038000
+    function deposit(uint256 _id, uint256 _assets, address _receiver) external;
```

Comment:
Who is supposed to call `deposit`? Is it the vault manager or the Fortress user? I would have thought the user deposits into the managed vault, but then the user should not be concerned about the Y2K vault ID.

Reply:
The manager of the AssetVault (which initially is the manager of the MetaVault, can be updated).